schemas/org.whatwg/media_element/jsonschema/1-0-0

```diff
@@ -0,0 +1,265 @@
+      ],
+      "description": "The media file format",
+      "maxLength": 255,
+      "minLength": 1
+    },
+    "fullscreen": {
+      "type": [
+        "boolean",
+        "null"
+      ],
+      "description": "If the video element is fullscreen"
+    },
+    "mediaType": {
+      "type": "string",
+      "description": "If the media is a video element, or audio",
+      "enum": [
+        "AUDIO",
+        "VIDEO"
+      ],
+      "maxLength": 5
+    },
+    "networkState": {
+      "description": "The current state of the fetching of media over the network",
+      "enum": [
+        "NETWORK_EMPTY",
+        "NETWORK_IDLE",
+        "NETWORK_LOADING",
+        "NETWORK_NO_SOURCE"
+      ],
+      "type": "string"
+    },
+    "pictureInPicture": {
+      "description": "If the video element is showing Picture-In-Picture",
+      "type": [
+        "boolean",
+        "null"
+      ]
+    },
+    "played": {
+      "type": [
+        "array",
+        "null"
+      ],
+      "description": "An array of time ranges played",
+      "items": {
+        "type": "object",
+        "description": "A time range",
+        "properties": {
+          "start": {
+            "type": "number",
+            "description": "The beginning of the time range",
+            "minimum": 0,
+            "maximum": 9007199254740991
+          },
+          "end": {
+            "type": "number",
+            "description": "The end of the time range",
+            "minimum": 0,
+            "maximum": 9007199254740991
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "htmlId": {
+      "type": "string",
+      "description": "The HTML id of the element",
+      "maxLength": 65535
+    },
+    "preload": {
+      "type": "string",
+      "description": "The 'preload' HTML attribute of the media",
+      "maxLength": 65535
+    },
+    "readyState": {
+      "type": "string",
+      "description": "The readiness of the media",
+      "enum": [
+        "HAVE_NOTHING",
+        "HAVE_METADATA",
+        "HAVE_CURRENT_DATA",
+        "HAVE_FUTURE_DATA",
+        "HAVE_ENOUGH_DATA"
+      ]
+    },
+    "seekable": {
+      "type": "array",
+      "description": "Seekable time range(s)",
+      "items": {
+        "type": "object",
+        "description": "A time range",
+        "properties": {
+          "end": {
+            "description": "The end of the time range",
+            "maximum": 9007199254740991,
+            "minimum": 0,
+            "type": "number"
+          },
+          "start": {
+            "description": "The beginning of the time range",
+            "maximum": 9007199254740991,
+            "minimum": 0,
+            "type": "number"
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "seeking": {
+      "type": "boolean",
+      "description": "If the media is in the process of seeking to a new position"
+    },
+    "src": {
+      "type": "string",
```

Comment:
I think `src` can be null if using:

```
  <video id="playerid" playsinline controls>
    <source src="test-video.mp4" type="video/mp4" />
  </video>
```

`currentSource`, or currentSrc in the case of the HTML API, likely won't ever be null as far as I can tell, so that should probably be the required field

Reply:
'src' has been removed from required